package.json

```diff
@@ -43,22 +43,28 @@
     "clean-webpack-plugin": "0.1.10",
     "colors": "1.1.2",
     "compression": "1.5.2",
+    "connect-ensure-login": "^0.1.1",
     "cookie-parser": "1.4.3",
     "css-loader": "0.28.7",
     "ejs": "2.5.5",
     "express": "4.13.1",
+    "express-session": "^1.15.6",
     "extract-text-webpack-plugin": "2.1.2",
     "immutable": "3.8.1",
     "jsonwebtoken": "7.1.9",
+    "jwt-decode": "^2.2.0",
```

Comment:
Can we remove jwt-decode? I think you are using jsonwebtoken?

Reply:
D'oh. Yeah, of course. My bad. Good catch!